src/pages/Login/Login.js

```diff
@@ -1,7 +1,90 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
+import Sns from './Sns';
+
 const Login = () => {
-  return <div />;
+  const [person, setPerson] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setPerson({ ...person, [name]: value });
+  };
+  // 아이디, 비밀번호 정규식
+  const passwordRegExp =
+    /^(?=.*[a-z])(?=.*\d)(?=.*[$@$!%*#?&])[a-z\d$@$!%*#?&]{8,16}$/;
+  const idRegExp = /^[a-z]+[a-z0-9]{4,12}$/;
+
+  const validation = idRegExp.test(person.id) && passwordRegExp.test(person.pw);
+
+  const navigate = useNavigate();
+
+  const signIn = e => {
+    e.preventDefault();
+    fetch('http://10.58.5.132:8000/users/login', {
+      method: 'POST',
+      body: JSON.stringify({
+        username: person.id,
+        password: person.pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.message === 'SUCCESS') {
+          navigate('/');
+          alert('로그인 성공!');
+        }
+      });
+  };
+
+  return (
+    <div>
```

Comment:
전체를 감싸고 있는 최상위 태그에 className을 부여하시고 nesting해주셔야 다른 컴포넌트의 스타일링에 영향을 미치지 않을 수 있습니다!
```suggestion
    <div className="login">
```